search/lib/cmr.js

```diff
@@ -61,6 +61,11 @@ async function findGranules (params = {}) {
   return response.data.feed.entry;
 }
 
+async function getProviders () {
+  const rawProviders = await axios.get(makeCmrSearchUrl('/ingest/providers'));
```

Comment:
When I hit the root endpoint, it's giving me a 404 error, and I think this might be the cause. `makeCMrSearchUrl` is using `settings.cmrSearchHost`, which is `cmr.earthdata.nasa.gov/search`, then adding on `/ingest/providers` making the full link
`cmr.earthdata.nasa.gov/search/ingest/providers`. 
But the URL it needs to hit to get the providers is 
`cmr.earthdata.nasa.gov/ingest/providers`.